doc/ref_arch/openstack/chapters/chapter02.rst

```diff
@@ -427,8 +435,10 @@ Cloud Infrastructure Hardware Profile Requirements (source `RM 5.4 <../../../ref
 Table 2-4a: Reference Model Requirements - Cloud Infrastructure Hardware
 Profile Requirements
 
-Cloud Infrastructure Hardware Profile-Extensions Requirements (source `RM 5.4 <../../../ref_model/chapters/chapter05.md#54-cloud-infrastructure-hardware-profiles-features-and-requirements>`__)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Cloud Infrastructure Hardware Profile-Extensions Requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+(source :ref:`ref_model/chapters/chapter05:cloud infrastructure software profiles features and requirements`)
```

Comment:
@collivier error, it should be:
:ref:`ref_model/chapters/chapter05:cloud infrastructure hardware profiles features and requirements.`